clouds/databricks/modules/Makefile

```diff
@@ -34,16 +61,20 @@ test: check check-extra venv
 	done; \
 	deactivate
 
-remove: check
+remove: venv3 check
 	echo "Removing modules..."
-	python3 $(COMMON_DIR)/python_utils/create_drop_functions.py
+	$(VENV3_BIN)/python  $(COMMON_DIR)/python_utils/create_drop_functions.py $(SQL_DIR) $(BUILD_DIR)/drop.sql $(DB_SCHEMA)
 	databricks workspace mkdirs $(SQL_DEPLOY_PATH)
-	databricks workspace import --overwrite --language SQL $(SQL_PATH) $(SQL_DEPLOY_PATH)/dropUDF
+	databricks workspace import --overwrite --language SQL $(BUILD_DIR)/drop.sql $(SQL_DEPLOY_PATH)/drop.sql
```

Comment:
we can maybe call the output `DROP_FUNCTIONS.sql` as in other providers.

Reply:
Yes, @Jesus89 also told me about it, we decided to do it in a separate ticket (my initial attempt to list functions using information_schema failed, we'll need to figure out how to do it)

Reply:
Yes, let's put that aside from now.